pkg/cli/admin/release/new.go

```diff
@@ -810,6 +810,24 @@ func resolveImageStreamTagsToReferenceMode(inputIS, is *imageapi.ImageStream, re
 				continue
 			}
 
+			statusRef := findStatusTagEvents(inputIS.Status.Tags, ref.Name)
+
+			generation := int64(0)
```

Comment:
nit: I think you can drop this and use `generation := *ref.Generation` inside the subsequent `if`.